snap/hooks/remove

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env bash
```

Comment:
ok, the only problem I can see with this is that we are assuming bash, and bash may or may not be installed on the host, we should normalize on sh and remove bashisms

Reply:
Converted from bash to sh :heavy_check_mark: